src/ivc/fold_relaxed_plonk_instance_chip.rs

```diff
@@ -223,6 +345,12 @@ pub(crate) struct AssignedWitness<C: CurveAffine> {
     /// All params in this structure used as part of seeds for generate this challenge
     r: Vec<AssignedBit<C::Base>>,
 }
+impl<C: CurveAffine> ops::Deref for AssignedWitness<C> {
```

Comment:
what is the purpose of Deref here?

Reply:
The names of all the variables do not overlap, so for the sake of brevity I made Deref so that I don't have to write `w.relaxed_plonk_instance` every time I access fold* variables